firmware/config/boards/microrusefi/openblt/STM32F429ZITx_FLASH.ld

```diff
@@ -0,0 +1,222 @@
+/*
```

Comment:
.ld files are not board specific but MCU specific right? let's move out of MRE folder

Reply:
Done.